playground.txt

```diff
@@ -0,0 +1,56 @@
+\c nc_games_test
```

Comment:
it's fine to have a playground but I wouldn't recommend commiting the files to the repo. You can git ignore them if you won't remember to avoid adding them to your commit.